### PR TITLE
samples: bluetooth: mesh: Add nrf54l05 and nrf54l10 targets

### PR DESCRIPTION
**Pull request by m-alperen-sener**
Adding supported targets to mesh samples.
Disabling duplicate test scenarios from twister.

### CODE REVIEW

**Review by PavelVPV — APPROVED** [review]

**Review by Balaklaka — CHANGES_REQUESTED** [review]
There should be something in release notes about adding nrf54l10/05 under Bluetooth Mesh samples.

File: scripts/quarantine_integration.yaml [hunk 1]
```diff
@@ -161,12 +163,23 @@
     - nrf5340dk/nrf5340/cpuapp
     - nrf5340dk/nrf5340/cpuapp/ns
     - thingy53/nrf5340/cpuapp
+    - nrf54l15dk/nrf54l10/cpuapp
```
[the remark this answers is not in the document]
Comment (m-alperen-sener) [reply scripts/quarantine_integration.yaml:166]: > nrf52840dk/nrf52840 is in this list.

the board is tested through the dfu variant.

> Why is not nrf54l15dk/nrf54l15/cpuapp not in the list? There is more of the same other places.

I think when the board first added it is not restricted on integration for any sample. We can consider it now.

**Comment by m-alperen-sener** [comment]
> There should be something in release notes about adding nrf54l10/05 under Bluetooth Mesh samples.

Requested here:
https://github.com/nrfconnect/sdk-nrf/pull/19240#discussion_r1876041596

**Review by Balaklaka — APPROVED** [review]

**Review by ardo-nordic — APPROVED** [review]
Looks alright to me, except for somewhat inconsistent formatting of platforms lists in yaml files, which isn't a big deal I guess.

**Review by jukkar — APPROVED** [review]

**Pull request merged**